mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.java

```diff
@@ -176,7 +176,7 @@ public View onCreateView(LayoutInflater inflater,
         // Inflate the layout for this fragment
         Log.i(TAG, "onCreateView");
         Log.d(TAG, "isAdded = " + isAdded());
-        return inflater.inflate(R.layout.openhabwidgetlist_fragment, container, false);
+        return inflater.inflate(R.layout.widgetlist_fragment, container, false);
```

Comment:
Maybe name the layout fragment_widgetlist for consistency?

Reply:
Right, I've overseen it.